mod_reforged/hooks/ai/tactical/behavior.nut

```diff
@@ -3,6 +3,13 @@
     {
         // Currently Mortars are near invincible and not meant to be killed. But AI still targets them sometimes. This line should make it so they are basically never targeted.
         if (_target.getType() == ::Const.EntityType.Mortar) return 0.01;
-        return __original(_entity, _target, _skill);
+
+        local score = __original(_entity, _target, _skill);
+        if (_skill != null && !_skill.isRanged() && _target.getSkills().hasSkill("effects.rf_rebuke") && _target.getTile().getDistanceTo(_entity.getTile()) == 1)
```

Comment:
This does not check all conditions whether Rebuke would trigger.
According to the effect it doesn't trigger on stuns or while Riposte is active

Maybe it's time to refactor the rebuke effect to have some sort of `willProcc` function. This would run over all conditions and return true or false. Then it can be used within the rebuke effect aswell as from the outside from the AI code

Reply:
Good point. I've added it.